script/get-operator-versions.sh

```
@@ -0,0 +1,98 @@
+
+    if [[ -z "$version" ]]; then
+        return
+    fi
+
+    [[ "$version" != v* ]] && version="v${version}"
+    echo "$version"
+}
+
+# Fetch all versions in parallel
+tmpdir=$(mktemp -d)
+trap "rm -rf $tmpdir" EXIT
```

Shellcheck (0.11.0)

[warning] Use single quotes, otherwise this expands now rather than when signalled.

(SC2064)